frontend/src/container/Sidebar.jsx

```diff
@@ -128,41 +127,47 @@ export class Sidebar extends React.Component {
     return !unLoggedAllowedPageList.some(url => props.location.pathname.startsWith(url))
   }
 
+  hasClass = (element, className) => {
```

Comment:
rename the function to something more explicit and/or add comment explaining it

Reply:
fixed (did both)